src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -76,14 +104,14 @@ class P5VisualizerTemplate extends P5Visualizer(paramDesc) {
         super(seq)
```

Comment:
Not sure how this remained here -- it's not actually good practice to define a constructor that only calls the super constructor, as that is the default behavior for a JavaScript class. I recommend taking out the constructor as long as we are overhauling this example, which then also lets you get rid of the import of the type GenericParamDescription, which is pretty esoteric.

Reply:
ok, I've done this.... but I can't test this visualizer (even on the main ui2 branch) because workbench isn't working right

Reply:
You can test it by commenting out the "export module" line of Grid.ts. (You're just not going to be able to commit a version with that line commented out because of all of our pre-commit checks.) If you don't want to bother, rest assured I will test it before merging.